assets/templates/feedback.tmpl

```diff
@@ -23,23 +23,34 @@
                     <fieldset>
                         <legend class="font-size--16"><strong>What's it to do with?</strong></legend>
                         <div class="margin-top--2 margin-bottom--4">
+                            {{ if .PreviousURL }} 
                             <div class="multiple-choice margin-top--1">
                                 <input type="radio" class="multiple-choice__input" id="specific-page" name="type" value="A specific page" checked>
                                 <label class="multiple-choice__label font-size--16" for="specific-page">
                                     This specific page
-                                    &nbsp; &nbsp;<span class="form-helper">(...{{.Metadata.Description}})</span>
                                     <input type="hidden" value="{{.PreviousURL}}" name="url">
                                 </label>
+                            <div class="multiple-choice__content height--5">
```

Comment:
A small one. Just indent (by tabbing) this entire `div` so that it is clear that this `div` is part of `div` mentioned in `line 27`